Tests/configure_and_test_integration_instances.py

```diff
@@ -230,6 +231,11 @@ def get_new_and_modified_integrations(git_sha1):
         isinstance(file_path, str) and checked_type(file_path, all_integration_regexes)
     ]
 
+    integrations_name_to_path_dict = dict([
+        (filepath_to_integration_name(file_path), file_path)
+        for file_path in (added_integration_files + modified_integration_files)
+    ])
+
```

Comment:
I am not sure if we need to create this data structure. lets check if we can check the path when we need it without crating this dict.